Packs/AccentureCTI/.pack-ignore

```diff
@@ -7,4 +7,11 @@ uuid
 CustomCVE
 acti
 getThreatIntelReport
-get-fundamentals-by-uuid
\ No newline at end of file
+get-fundamentals-by-uuid
+
+# Temporarily ignoring GR103.
+# For more details, refer to the ticket: https://jira-dc.paloaltonetworks.com/browse/CIAC-11656
+[file:layoutscontainer-ACTI_Intelligence_Report.json]
+ignore=GR103
+[file:layoutscontainer-ACTI_Intelligence_Alert.json]
+ignore=GR103
```

Comment:
Consider the following format: 

```suggestion
[file:layoutscontainer-ACTI_Intelligence_Report.json]
ignore=GR103 # temporary, see CIAC-11656
[file:layoutscontainer-ACTI_Intelligence_Alert.json]
ignore=GR103 # temporary, see CIAC-11656
```

advantages: 
- comment is linked to the ignore code 
- nobody can mess it by mistake
- no need to remember to remove the two comments before it
 
in places where we ignore two codes, 
```python
ignore=GR103, ZZ000 # GR103 is temporary, see CIAC-11656
```